docs/source/daml/code-snippets/Keys.daml

```diff
@@ -0,0 +1,203 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Since I was wondering about this: Yes this is tested, we test everything in that directory.

Reply:
That was one of my main motivations for this. It would be even better if we had script-based tests for things like this as well.

Reply:
Wouldn’t be hard to make a directory that contains a bunch of scripts that are tested via `daml test-script`.